vulkan_generator/vulkan_utils/parsing_utils.py

```diff
@@ -122,3 +130,44 @@ def try_get_attribute_as_list(elem: ET.Element, attrib: str) -> Optional[List[st
 def clean_type_string(string: str) -> str:
     """Cleans the string from whitespace and ',' and ');'"""
     return string.replace(os.linesep, "").replace(" ", "").replace(",", "").replace(");", "")
+
+################################
+#                              #
+#          Enum Utils          #
+#                              #
+################################
+
+
+@dataclass
+class EnumFieldRepresentation:
+    value: int
+    representation: str
+
+
+def get_enum_field_from_value(value_str: str) -> EnumFieldRepresentation:
+    representation = value_str
+    value = int(representation, 0)
+
+    return EnumFieldRepresentation(value=value, representation=representation)
+
+
+def get_enum_field_from_bitpos(bitpos_str: str, bit64: bool) -> EnumFieldRepresentation:
+    bitpos = int(bitpos_str)
+    value = 1 << bitpos
+    representation = f"0x{value:08x}"
```

Comment:
Nit: As an if-else this might be a little more readable

Reply:
Hmm.. there is nothing that requires if there so I assume your comment is about the if below(bit64). If not please let me know. 

I thought about it as well but in that case we need to repeat the `f"0x{value:08x}"` twice and it felt error prone to me. This way it just ads `ULL` at the end based on if. If you feel strong about it, you can write your suggestion.